<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerMapWidgetBase</class>
 <widget class="QWidget" name="QgsComposerMapWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>483</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Map options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>230</width>
        <height>395</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Map</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="mPreviewModeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="mUpdatePreviewButton">
         <property name="text">
          <string>Update preview</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel1">
           <property name="text">
            <string>Width</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mWidthLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="mWidthLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Height</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mHeightLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="mHeightLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Scale</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mScaleLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="mScaleLineEdit">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="mRotationLabel">
           <property name="text">
            <string>Rotation</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="mRotationSpinBox">
           <property name="suffix">
            <string> degrees</string>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QCheckBox" name="mKeepLayerListCheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Lock layers for map item</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QCheckBox" name="mDrawCanvasItemsCheckBox">
           <property name="text">
            <string>Draw map canvas items</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="mOverviewFrameMapComboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="mOverviewFrameMapLabel">
         <property name="text">
          <string>Overview frame</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="mFrameStyleLabel_2">
         <property name="text">
          <string>Frame style</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="mOverviewFrameStyleButton">
         <property name="text">
          <string>Change...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>230</width>
        <height>395</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Extents</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="mXMinLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>X min</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mXMinLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mXMinLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="mXMaxLabel">
           <property name="text">
            <string>X max</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mXMaxLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="mXMaxLineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="mYMinLabel">
           <property name="text">
            <string>Y min</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mYMinLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="mYMinLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="mYMaxLabel">
           <property name="text">
            <string>Y max</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>mYMaxLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="mYMaxLineEdit"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="mSetToMapCanvasExtentButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Set to map canvas extent</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>218</width>
        <height>716</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Grid</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="mGridCheckBox">
         <property name="title">
          <string>Show grid?</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QLabel" name="mGridTypeLabel">
            <property name="accessibleName">
             <string extracomment="Hello translotor"/>
            </property>
            <property name="text">
             <string>Grid &amp;type</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mGridTypeComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="mGridTypeComboBox"/>
          </item>
          <item row="1" column="0" colspan="4">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="mIntervalXLabel">
              <property name="text">
               <string>Interval X</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mIntervalXSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="mIntervalXSpinBox">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>9999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="mIntervalYLabel">
              <property name="text">
               <string>Interval Y</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mIntervalYSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="mIntervalYSpinBox">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>9999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="mOffsetXLabel">
              <property name="text">
               <string>Offset X</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mOffsetXSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="mOffsetXSpinBox">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>9999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="mOffsetYLabel">
              <property name="text">
               <string>Offset Y</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mOffsetYSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="mOffsetYSpinBox">
              <property name="decimals">
               <number>5</number>
              </property>
              <property name="maximum">
               <double>9999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="mCrossWidthLabel">
              <property name="text">
               <string>Cross width</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mCrossWidthSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="mCrossWidthSpinBox">
              <property name="decimals">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="mLineWidthLabel">
              <property name="text">
               <string>Line width</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mLineWidthSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="mLineWidthSpinBox">
              <property name="decimals">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="mFrameStyleLabel">
              <property name="text">
               <string>Frame style</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="mFrameStyleComboBox"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="mFrameWidthLabel">
              <property name="text">
               <string>Frame width</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QDoubleSpinBox" name="mFrameWidthSpinBox"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="mLineColorLabel">
              <property name="text">
               <string>Line color</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mLineColorButton</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsColorButton" name="mLineColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="mDrawAnnotationCheckBox">
            <property name="text">
             <string>Draw annotation</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="mAnnotationPositionLabelLeft">
              <property name="text">
               <string>Annotation position left side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mAnnotationPositionLeftComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="mAnnotationPositionLeftComboBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="mAnnotationPositionLabelRight">
              <property name="text">
               <string>Annotation position right side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mAnnotationPositionRightComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="mAnnotationPositionRightComboBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="mAnnotationPositionLabelTop">
              <property name="text">
               <string>Annotation position top side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mAnnotationPositionTopComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="mAnnotationPositionTopComboBox"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="mAnnotationPositionLabelBottom">
              <property name="text">
               <string>Annotation position bottom side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mAnnotationPositionRightComboBox</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="mAnnotationPositionBottomComboBox"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="mAnnotationDirectionLabelLeft">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>Annotation direction left side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mAnnotationDirectionComboBoxLeft</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="mAnnotationDirectionComboBoxLeft"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="mAnnotationDirectionLabelRight">
              <property name="text">
               <string>Annotation direction right side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="buddy">
               <cstring>mAnnotationDirectionComboBoxRight</cstring>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="mAnnotationDirectionComboBoxRight"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="mAnnotationDirectionLabelTop">
              <property name="text">
               <string>Annotation direction top side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="mAnnotationDirectionComboBoxTop"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="mAnnotationDirectionLabeBottom">
              <property name="text">
               <string>Annotation direction bottom side</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QComboBox" name="mAnnotationDirectionComboBoxBottom"/>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="mAnnotationFontButton">
            <property name="text">
             <string>Font...</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="mDistanceToFrameLabel">
            <property name="text">
             <string>Distance to map frame</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mDistanceToMapFrameSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="2" colspan="2">
           <widget class="QDoubleSpinBox" name="mDistanceToMapFrameSpinBox"/>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QLabel" name="mCoordinatePrecisionLabel">
            <property name="text">
             <string>Coordinate precision</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mCoordinatePrecisionSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="2" colspan="2">
           <widget class="QSpinBox" name="mCoordinatePrecisionSpinBox"/>
          </item>
          <item row="8" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mDrawAnnotationCheckBox</tabstop>
  <tabstop>mAnnotationFontButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
